tests/e2e/Services/Account/AccountCustomClientTest.php

```diff
@@ -2251,8 +2251,11 @@ public function testPhoneVerification(array $data): array
 
         $smsRequest = $this->getLastRequest();
 
+        $message = $smsRequest['data']['message'];
+        $token = substr($message, 0, 6);
+
         return \array_merge($data, [
-            'token' => $smsRequest['data']['secret']
+            'token' => $token
```

Comment:
Not sure if this will solve the problem entirely, seems to be a race condition between the webhook firing and the message sending. No action just noting

Reply:
I did var dump and there was no secret, but there was message `XXXXXX is your code for bla bla bla`

Reply:
Unless this is larger problem, it was just a leftover after Khushboo's change to improve copy